src/styles.css

```
@@ -101,6 +101,33 @@
 	}
 }
 
+/* Primary call to action — one rose accent, sharp corners. */
+@layer components {
+	[data-cta] {
+		cursor: pointer;
+
+		display: inline-flex;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)